Bookish/Controllers/CatalogController.cs

```diff
@@ -114,10 +131,16 @@ public IActionResult BookList()
         }
         return View(bvm);
     }
-
-    /*public IActionResult SortList(BookViewModel modelToSort)
+    
+    public IActionResult BorrowerList()
     {
-        return View(modelToSort);
-    }*/
+        BorrowerViewModel bvm = new BorrowerViewModel();
+        using (var context = new LibraryContext())
+        {
+            bvm.EntryQuery = context.Borrowers.Include(b => b.CopyList).Include(b => b.BorrowList);
+            bvm.CatalogEntries = bvm.EntryQuery.ToList();

```

Comment:
I'm sure I made this comment yesterday... but I don't think the `EntryQuery` was used anywhere else. If you just want a list of catalog entries, this can just be 
```
bvm.CatalogEntries = context.Borrowers.Include(b => b.CopyList).Include(c => c.BorrowList).ToList();
```
also, be wary of using the same variable name in the two anonymous functions (you used `b` twice)

If you do it this way and aren't otherwise using `EntryQuery`, you can remove it from the `BorrowerViewModel`